modules/ROOT/pages/syntax-quick-reference.adoc

```diff
@@ -0,0 +1,1290 @@
+[%collapsible.result]
+====
+include::macros:example$ui.adoc[tag=menu]
+====
+
+.Button macro
+[#ex-button]
+----
+include::macros:example$ui.adoc[tag=button]
+----
+
+.View result of <<ex-button>>
+[%collapsible.result]
+====
+include::macros:example$ui.adoc[tag=button]
+====
+
+== Attributes and substitutions
+
+.Attribute declaration and usage
+[#ex-attributes]
+----
+:url-home: https://asciidoctor.org
+:link-docs: https://asciidoctor.org/docs[documentation]
+:summary: Asciidoctor is a mature, plain-text document format for \
+       writing notes, articles, documentation, books, and more. \
+       It's also a text processor & toolchain for translating \
+       documents into various output formats (i.e., backends), \
+       including HTML, DocBook, PDF and ePub.
+:checkedbox: pass:normal[{startsb}&#10004;{endsb}]
+
+Check out {url-home}[Asciidoctor]!
+
+{summary}
+
+Be sure to read the {link-docs} too!
+
+{checkedbox} That's done!
+----
+
+.View result of <<ex-attributes>>
+[%collapsible.result]
+====
+// I have to use a nested doc hack here, otherwise the attributes won't resolve
+[.unstyled]
+|===
+a|
+:url-home: https://asciidoctor.org
+:link-docs: https://asciidoctor.org/docs[documentation]
+:summary: Asciidoctor is a mature, plain-text document format for \
+       writing notes, articles, documentation, books, and more. \
+       It's also a text processor & toolchain for translating \
+       documents into various output formats (i.e., backends), \
+       including HTML, DocBook, PDF and ePub.
+:checkedbox: pass:normal[{startsb}&#10004;{endsb}]
+
+Check out {url-home}[Asciidoctor]!
+
+{summary}
+
+Be sure to read the {link-docs} too!
+
+{checkedbox} That's done!
+|===
+====
+
+.Attribute assignment precedence (highest to lowest)
+- Attribute passed to the API or CLI that does not end in `@`
+- Attribute defined in the document
+- Attribute passed to the API or CLI that ends in `@`
+- Intrinsic attribute value (default values)
+
+TIP: To make an attribute value that is passed to the API or CLI have a lower precedence than an assignment in the document, add an `@` symbol to the end of the attribute value.
+
+To learn more about the available attributes and substitution groups see:
+
+* xref:attributes:document-attributes-reference.adoc[]
+* xref:attributes:character-replacement-reference.adoc[]
+* xref:attributes:intrinsic-attributes-reference.adoc[]
+* xref:subs:apply-subs-to-blocks.adoc#subs-groups[Substitution Groups]
+
+.Counter attributes
+[#ex-counter]
+----
+include::attributes:example$counter.adoc[tag=base]
+----
+
+.View result of <<ex-counter>>
+[%collapsible.result]
+====
+[caption="Table 1. "]
+include::attributes:example$counter.adoc[tag=base]
+====
+
+== Text replacements
+
+include::subs:partial$subs-symbol-repl.adoc[]
+
+Any named, numeric or hexadecimal {url-char-xml}[XML character reference] is supported.
+
+== Escaping substitutions
+
+.Backslash
+[#ex-slash]
+----
+include::subs:example$subs.adoc[tag=backslash]
```

Comment:
Maybe we should include an example where we escape unconstrained formatting marks:

```
Beatle\\**mania**
```

Reply:
I'm nervous about getting into the depths of escaping here because it has so many edge cases. It would be better to link out to the language reference to address these scenarios.